editions/tw5.com/tiddlers/wikitext/Formatting in WikiText.tid

```diff
@@ -1,19 +1,38 @@
 caption: Formatting
-created: 20131205155959399
-modified: 20220513120211686
+created: 20220727060921261
+modified: 20220821183525017
 tags: WikiText
 title: Formatting in WikiText
 type: text/vnd.tiddlywiki
 
-Available character formatting includes:
+Available character formatting in WikiText includes:
 
-* <code>&#96;backticks&#96;</code> for `code` (<<.icon $:/core/images/mono-line>>)
-** Alternatively, <code>&#96;&#96;double backticks allows &#96;embedded&#96; backticks&#96;&#96;</code><br><br>
-* `''bold''` for ''bold text'' (<<.icon $:/core/images/bold>>)<br><br>
-* `//italic//` for //italic text// (<<.icon $:/core/images/italic>>)<br><br>
-* `__underscore__` for __underscored text__ (<<.icon $:/core/images/underline>>)<br><br>
-* `^^superscript^^` for ^^superscripted^^ text (<<.icon $:/core/images/superscript>>)<br><br>
-* `,,subscript,,` for ,,subscripted,, text (<<.icon $:/core/images/subscript>>)<br><br>
-* `~~strikethrough~~` for ~~strikethrough~~ text (<<.icon $:/core/images/strikethrough>>)
+<div class="no-code-border">
+
+|Wikitext | Button |Win Shortcut | Mac |Rendered Output |h
+|2 single hyphens are used for `''bold text''`| <kbd><<.icon $:/core/images/bold>></kbd> | <kbd>ctrl-B</kbd> |<kbd>win-B</kbd> |2 single hyphens are used for ''bold text''|
+|2 slashes are used for `//italic text//`| <kbd><<.icon $:/core/images/italic>></kbd> | <kbd>ctrl-I</kbd> |<kbd>win-I</kbd> |2 slashes are used for //italic text//|
+|2 underscores are used for `__underscored text__`| <kbd><<.icon $:/core/images/underline>></kbd> | <kbd>ctrl-U</kbd> |<kbd>win-U</kbd> |2 underscores are used for __underscored text__ |
+|2 circumflex accents are used for `^^superscripted^^` text | <kbd><<.icon $:/core/images/superscript>></kbd> | <kbd>ctrl-shift-P</kbd> |<|2 circumflex accents are used for ^^superscripted^^ text |
+|2 commas are used for `,,subscripted,,` text | <kbd><<.icon $:/core/images/subscript>></kbd> | <kbd>ctrl-shift-B</kbd> |<|2 commas are used for ,,subscripted,, text |
+|2 tilde signs are used for `~~strikethrough~~` text | <kbd><<.icon $:/core/images/strikethrough>></kbd> | <kbd>ctrl-T</kbd> |<|2 tilde signs are used for ~~strikethrough~~ text |
+|1 backtick is used for ```code` `` | <kbd><<.icon $:/core/images/mono-line>></kbd> | <kbd>ctrl-M</kbd> |<|1 backtick is used for `code` |
+
+''Special case to show embedded backticks''
+
+|Wikitext |Rendered Output|h
+|<code>&#96;&#96;double backticks allows &#96;embedded&#96; backticks&#96;&#96;</code>|``double backticks allows `embedded` backticks``|
+
+''For those who are interested''
```

Comment:
We don't phrase things like that. I've mentioned this before: these are reference documents. They are not supposed to be idiomatic and colloquial, they are supposed to formal and crisp.

Better to say "Further Information" or "Advanced Topics".